matsim/src/main/java/org/matsim/core/events/EventsManagerImpl.java

```diff
@@ -108,6 +108,11 @@ protected HandlerInfo(final Class<?> eventClass, final EventHandler eventHandler
 			this.eventHandler = eventHandler;
 			this.method = method;
 		}
+
+		@Override
+		public String toString() {
+			return String.format("eventClass: %s, eventHandler: %s", eventClass.toString(), eventHandler.toString());
```

Comment:
include the method as well

Reply:
When I include the `method` it becomes too long. One can derive what is the method from `eventHandler `